README.md

```diff
@@ -0,0 +1,72 @@
+# Etna
+
+## Local development
+
+### Docker
+
+Etna uses Docker to create a consistent environment for local development.
+
+On macOS and Windows, Docker requires [Docker
+Desktop](https://www.docker.com/products/docker-desktop) to be installed. Linux
+users should install the Docker engine using their distribution's package
+manager or [download a `.deb` or
+`.rpm`](https://docs.docker.com/engine/install/)
+
+Once installed, we need to build our containers. We use
+[`docker-compose`](https://docs.docker.com/compose/) to orchestrate the
+building of the project's containers, one for each each service:
+
+#### `db`
+
+The database service built from the official [postgres](https://hub.docker.com/_/postgres/) image
+
+#### `elasticsearch`
+
+The search backend, build from the official [elasticsearch](https://hub.docker.com/_/elasticsearch/) image
+
+#### `web`
+
+Our custom container responsible for running the application. Built from the
+official [python 3.9](https://hub.docker.com/_/python/) base image
+
+During the build step our custom container installs our project's dependencies
+using [Poetry](https://python-poetry.org) as defined in
+[`pypproject.toml`](pyproject.toml) using the approprpriate versions outlined
+in [`Poetry.lock`](Poetry.lock)
+
+Build steps for the `web` container are defined in the project's [`Dockerfile`](Dockerfile).
+
+Our `web` container
+[depends](https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on)
+on both the `db` and `elasticsearch` containers in order for the `web`
+container to communicate with those services.
+
+### Getting started
+
+#### Docker
+
+##### Build containers
+
+`docker-compose build`
```

Comment:
A `.env` file is required for this to work.
Can you provide an example file?

Reply:
Good spot!

I hadn't pushed that last commit (a743a76)